easybuild/easyblocks/m/msm.py

```diff
@@ -98,6 +110,22 @@ def build_step(self):
 
     def make_installdir(self):
         """Override installdir creation"""
+
         self.log.warning("Not removing installation directory %s" % self.installdir)
         self.cfg['keeppreviousinstall'] = True
         super(EB_MSM, self).make_installdir()
+
+    def install_step(self):
+        """Define files to be copied at installation."""
+
+        self.cfg['files_to_copy'] = [(['src/MSM/%s' % f for f in self.binfiles], 'bin')]
```

Comment:
please use `os.path.join`:

```python
self.cfg['files_to_copy'] = [([os.path.join('src', 'MSM', f) for f in self.binfiles], 'bin')]
```

Also, aren't `doc` & co missing? How will those get installed?

Reply:
@boegel `doc` are installed automatically by `make install`. As a matter of fact the `doc` directory is already checked in `sanity_check_step`.

Reply:
Wait, `make install` installs the docs, but not the binaries?! 🤦‍♂

Reply:
Yup